package.json

```diff
@@ -1,6 +1,19 @@
 {
+  "name": "harness-deploy-action",
+  "description": "Deploy to Harness with GitHub Action",
+  "main": "index.js",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "jest",
+    "prepare": "ncc build index.js -o dist --source-map --license licenses.txt"
```

Comment:
As this is taken from the [template action provided by GitHub](https://github.com/actions/javascript-action/blob/main/package.json#L8), I'd rather keep it as is, because other actions will also use the same naming, so changing ours will get confusing if switching between projects.